sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeActionsUtilClient.java

```diff
@@ -1317,6 +1317,8 @@ private AnalyzeActionsResult toAnalyzeActionsResultLanguageApi(AnalyzeTextJobSta
         if (!CoreUtils.isNullOrEmpty(errors)) {
             for (Error error : errors) {
                 if (error != null) {
+                    LOGGER.info("error target: " + error.getTarget());
+                    LOGGER.info("error message: " + error.getMessage());
```

Comment:
why are these logged? Shouldn't the error be returned as part of the result object?

Reply:
These can be removed. I was trying to log more for the targeting error link.